try:
    print(1 + "1")
except Exception as ex:
    print(ex)